paranormal-pioneers/project/ui/root.py

```diff
@@ -0,0 +1,128 @@
+from kivy.app import App
+from kivy.event import EventDispatcher
+from kivy.properties import (
+    ObjectProperty, ListProperty, StringProperty,
+    NumericProperty, Clock, partial
+)
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.textinput import TextInput
+
+from project.core.path import Path
+from project.core.terminal import Terminal
+
+root = Path(__file__).parent.resolve()
```

Comment:
In other files, constants like this have been uppercased. It could also be useful moving that font to the constants file which already exists.